packages/@misk/simpleredux/tests/bundle.test.ts

```diff
@@ -0,0 +1,21 @@
+import { readFileSync } from 'fs'
```

Comment:
What are these?

Reply:
Takes a snapshot of the built artifact of `simpleredux` library and fails if that changes. It will be useful in follow up PRs where we do no-op changes and want to ensure that the deployed package hasn't changed API.